### ISSUE TITLE
Diktat useless rule suppress checker

### ISSUE THREAD

**Issue by sanyavertolet** [saveourtool/diktat#1530]
It might be a good idea to implement feature of detecting rule suppresses that could be removed e.g. this code:
```Kotlin
@Suppress("TYPE_ALIAS")
fun foo() {
    println("diktak")
}
```
would show warning as far as @Suppress("TYPE_ALIAS")` is unnecessary.

Moreover, fix feature might be also good.
